default_kms_key_arn.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python2
+import sys
+import json
+import boto3
+
+# setup connection to AWS
+client = boto3.client('kms')
+
+# get the default KMS key
+key_aliases = client.list_aliases()['Aliases']
+
+arn = None
+
+for alias in key_aliases:
+    if alias['AliasName'] == 'alias/aws/lambda':
+        arn = alias['AliasArn']
+
+if not arn:
+    json.dump({
```

Comment:
Is there any point printing this JSON when it errors?

Reply:
Probably not, I was just trying to prevent extra errors on Terraform's side.